### DISCUSSION TITLE
get editor instance in JavaScript

### DISCUSSION THREAD

**Discussion by skunkwerk** [coder/code-server#5351] [Q&A]
How do I get a reference to the editor's instance in Javascript?  I believe code-server uses the monaco editor, right?
I'm looking to write some Javascript to extract the value of the current text in the editor, and I believe with Monaco I can just do
editor.getValue()
but I need a reference to the current editor.

thanks

**Comment by jsjoeio** [comment]
Hmm...i think it might be better to ask on the [monaco repo](https://github.com/Microsoft/monaco-editor)!
